docs/user_guide.md

```diff
@@ -248,3 +249,71 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
+* **Queued:** The scanning task is scheduled but not executed yet.
+* **Scanning:** The scanning process is in progress.
+* **Error:** The scanning process failed to complete.
+* **Complete:** The scanning process was successfully completed.
+
+For the **'Not Scanned'** and **'Queued'** statuses, a text label with status information is shown. For the **'Scanning'**, a progress bar will be displayed.
+If an error occurred, you can click on the **'View Log'** link to view the related logs.
+![browse project](img/log_viewer.png)
+
+If the process was successfully completed, a result bar is created. The width of the different colored sections indicates the percentage of features with vulnerabilities for a particular severity level.
+* **Color Red:** **High** level of vulnerabilities
+* **Color Orange:** **Medium** level of vulnerabilities
+* **Color Yellow:** **Low** level of vulnerabilities
+* **Color Grey:** **Unknown** level of vulnerabilities
+* **Color Green:** **None** vulnerabilities
+![browse project](img/bar_chart.png)
+
+Move the cursor over the bar, a tooltip with summary report will be displayed. Besides showing the total number of features with vulnerabilities and the total number of features in the scanned image tag, the report also lists the counts of features with vulnerabilities of different severity levels. The completion time of the last analysis process is shown at the bottom of the tooltip.
+![browse project](img/summary_tooltip.png)
+
+Click on the tag name link, the detail page will be opened. Besides the information about the tag, all the vulnerabilities found in the last analysis process will be listed with the related information. You can order or filter the list by columns.
+![browse project](img/tag_detail.png)
+
+**NOTES: You can initiate the vulnerability analysis for tag at anytime you want as long as the status is not 'Queued' or 'Scanning'** 
+
+**Scan All**
+
+In the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**, click on the **'SCAN NOW'** button to start the analysis process for all the existing images. 
+
+**NOTES: The scanning process is executed via multiple concurrent asynchronous tasks. There is no guarantee on the order of scanning or the returned results.** 
+![browse project](img/scan_all.png)
+
+To avoid frequently triggering the resource intensive scanning process, the availability of the button is restricted. It can be only triggered once in a predefined period. The next available time will be displayed besides the button.
+![browse project](img/scan_all2.png)
+
+**Scheduled Scan by Policy** 
+
+You can set policies to control the vulnerability analysis process. Currently, two options are available in the policy:
+* **None:** No policy is scheduled.
+* **Daily:** Policy is activated daily. It means an analysis job is scheduled to execute at the specified time everyday. The scheduled job will scan all the images in Harbor.

```

Comment:
an analysis job is scheduled to execute
==>
an analysis job is scheduled to be executed